build.gradle

```diff
@@ -0,0 +1,25 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
```

Comment:
can't this go in the android directory? I'd like to avoid having to many files lying in repository root

Reply:
unfortunately some of the Gradle bits need to live in the root dir. Android Studio (an IDE based upon IntelliJ) treats the "project root" as the directory that contains the root `build.gradle` and `settings.gradle`. They _could_ live in the `android/` dir, but then AS would end up ignoring all of the cpp code - which would be extremely inconvenient for development.